Parsers/ASimUserManagement/Parsers/vimUserManagementMicrosoftSecurityEvent.yaml

```diff
@@ -0,0 +1,317 @@
+          | where (targetusername_has=='*' or (EventData has targetusername_has)) and 
+              (actorusername_has=='*' or (EventData has actorusername_has)) and
+              (targetdomain_has=='*' or (EventData has targetdomain_has)) and
+              (anydomain_has=='*' or (EventData has anydomain_has) or (EventData has anydomain_has))
+          | parse-kv EventData as 
+            (
+                TargetUserName:string,
+                TargetDomainName:string,
+                TargetSid:string,
+                SubjectUserSid:string,
+                AccountType:string,
+                SubjectLogonId:string,
+                SubjectDomainName:string,
+                SubjectUserName:string
+            ) 
+            with (regex=@'<Data Name="(\w+)">{?([^<]*?)}?</Data>')
+          | where (actorusername_has=='*' or (SubjectUserName has actorusername_has)) and
+              (targetdomain_has=='*' or (TargetDomainName has targetdomain_has)) and
+              (anydomain_has=='*' or (TargetDomainName has anydomain_has) or (SubjectDomainName has anydomain_has))
+          | project-rename 
+              ActorOriginalUserType = AccountType,
+              ActorSessionId = SubjectLogonId,
+              ActorUserId = SubjectUserSid,
+              GroupDomain = TargetDomainName,
+              GroupId = TargetSid,
+              GroupName = TargetUserName
+          | extend GroupName = iff (GroupDomain == "", GroupName, strcat (GroupDomain, "\\" ,GroupName))
+          | parse-kv EventData as 
+              (
+                  MemberName:string,
+                  MemberSid:string
+              ) 
+              with (regex=@'<Data Name="(\w+)">{?([^<]*?)}?</Data>')
+          | where (targetusername_has=='*' or (MemberName has targetusername_has))
+          | project-rename 
+              TargetUserId = MemberSid,
+              TargetUsername = MemberName
+          | project TimeGenerated, Type, EventID, Computer, _ResourceId, _ItemId, GroupId, GroupName, ActorUserId, SubjectDomainName, SubjectUserName, ActorOriginalUserType, ActorSessionId, TargetUsername, TargetUserId
+          | extend 
+              GroupIdType = iif(isnotempty(GroupId), "SID","")
+      )
+  | lookup EventIDLookup on EventID
+  | invoke _ASIM_ResolveDvcFQDN ("Computer")
+  | lookup UserTypeLookup on ActorOriginalUserType
+  | project-rename 
+      DvcId = _ResourceId,
+      EventUid = _ItemId
+  | extend 
+      ActorDomain = SubjectDomainName,
+      ActorUserIdType = iif(isnotempty(ActorUserId), "SID",""),
+      ActorUsername = iff (SubjectDomainName == "", SubjectUserName, strcat (SubjectDomainName, '\\', SubjectUserName)),
+      Dvc = DvcHostname,
+      DvcIdType = iff (DvcId == "", "", "AzureResourceID"),
+      DvcOs = "Windows",
+      EventCount = int(1),
+      EventEndTime = TimeGenerated,
+      EventOriginalType = tostring(EventID),
+      EventProduct = 'Security Events',
```

Comment:
Should EventProduct be 'Windows' ?

Reply:
In both Windows Authentication and Windows Files we used Security Events